docs/usage/input.md

```diff
@@ -42,6 +42,7 @@ The following sections describe the columns that can be used in every sample-she
 | column                  | type          | required        | default      | description                                                                                                               |
 |-------------------------|---------------|-----------------|--------------|---------------------------------------------------------------------------------------------------------------------------|
 | ``adaptive_sampling``   | ``file``      |                 |              | allowed file extensions: [``csv``]. for ``nanopore`` adaptive sampling experiments, used to filter `stop_receiving` reads | 
+| ``bed``                 | ``file``      |                 |              | Bed file to filter the called variats with. allowed file extensions: [``bed``]                                            |

```

Comment:
suggestions:
- move to https://molgenis.github.io/vip/usage/input/#columns instead of specifying for all workflows
- change description to `allowed file extensions: [``bed``]. filter variants overlapping with regions in bed file<sup>2</sup>`
  - add entry below table: `<sup>2</sup> bed regions are 0-based with half-open [start, stop) intervals`